native-symbol.js

```diff
@@ -0,0 +1,9 @@
+// Returns the native Symbol, if there is one.
```

Comment:
I think I wouldn't create this module, instead I would just refer to `global` in all modules.
It just serves a shortcut for `global.Symbol` which seems not perfectly justified.

Other issue with it, is that if we assign polyfill to `global` then it'll return a _polyfill_ and not _native_ symbol as name implies

Reply:
Ok, I can do that, but I was trying to keep the line count minimal. Using global everywhere will require a larger PR.

Reply:
On further reflection, I think I can do this with fewer line changes than I was originally thinking.